src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1169,7 +1169,7 @@ private bool IsValidOverrideReturnType(Symbol overridingSymbol, TypeWithAnnotati
                 location,
                 new FormattedSymbol(overridingParameter, SymbolDisplayFormat.ShortFormat));
 
-        internal static void CheckValidNullableMethodOverride<TArg>(
+        internal static bool CheckValidNullableMethodOverride<TArg>(
```

Comment:
Consider adding a doc comment for the return value.

Reply:
Doc'ed. Would like @cston to confirm the correctness of the doc comment.